roms_tools/setup/grid.py

```diff
@@ -881,15 +885,15 @@ def __repr__(self) -> str:
         attr_str = ", ".join(f"{k}={v!r}" for k, v in attr_dict.items())
         return f"{cls_name}({attr_str})"
 
-    def _create_horizontal_grid(self) -> xr.Dataset():
+    def _create_horizontal_grid(self) -> xr.Dataset:
         """Create the horizontal grid based on a Mercator projection and store it in the
         'ds' attribute.
 
         Parameters
         ----------
         None
 
-        Returns
+        Returns:
```

Comment:
I think we have some errant items, but we're on the "no colon" side. Did you mean to make the separator `-------` a single char shorter instead?